src/dom.ts

```diff
@@ -155,6 +160,8 @@ export class ElementHandle<T extends Node = Node> extends js.JSHandle<T> {
       result.x += point.x / 4;
       result.y += point.y / 4;
     }
+    result.x = (result.x * 100 | 0) / 100;
```

Comment:
Let's instead round at line 193, to handle `_offsetPoint` case as well.

Reply:
Done